java/client/src/main/java/glide/api/commands/GenericClusterCommands.java

```diff
@@ -251,6 +251,66 @@ public interface GenericClusterCommands {
      */
     CompletableFuture<Object[]> scan(ClusterScanCursor cursor);
 
+    /**
+     * Incrementally iterates over the keys in the Cluster.
```

Comment:
The doc is TLDR. I think you just copied it, but it is good to say what is the difference in the functions. Why/When a user should use this one?

Reply:
They don't change the documentation between String and GlideString.  I think they wanted to consolidate the two APIs at some point...

Reply:
Maybe...
But these 2 funcs have 1:1 docs and signatures - really confusing.
On another hand, doc can be changed later. Up to you.

Reply:
I'd like them to decide on how to change these docs...